gap/CompilerLogic.gi

```diff
@@ -410,6 +410,15 @@ CapJitAddLogicTemplate(
     )
 );
 
+CapJitAddLogicTemplate(
+    rec(
+        variable_names := [ "number1", "number2" ],
+        #variable_filters := [ IsBigInt, IsBigInt ],
+        src_template := "number2 ^ number1 <> 0",
```

Comment:
> @zickgraf: I am unable to write the correct logic template.

Try `not number2 ^ number1 = 0` here. For Julia, we rewrite `not a = b` to `a <> b`, but only in the output. So what you see in the output might actually not be what the syntax tree stores internally. Feel free to open an issue about this. I came across this myself recently and this certainly should be improved.

Reply:
I have already tried and it did not work.